packages/inventory/src/ducks/inventory.ts

```diff
@@ -17,17 +17,32 @@ export interface CustomProperties {
   'UNICEF section': string;
 }
 
+/** interface product for inventory */
+export interface Product {
+  uniqueId: number;
+  productName: string;
+  isAttractiveItem: boolean;
+  materialNumber: string;
+  availability: string;
+  condition: string;
+  appropriateUsage: string;
+  accountabilityPeriod: number;
+  photoURL: string;
+  serverVersion: number;
+}
+
```

Comment:
I think this duplicates the Product Catalogue interface, is there a reason we are not using that.

Reply:
@p-netm I've confirmed they are the same..I'll remove this one